### PR TITLE
@storybook/angular add autoprefixing like angular-cli

### PR DESCRIPTION
**Pull request by alexan**
Issue: #5436 

@angular/storybook does not autoprefix like angular-cli. As described in this [issue](https://github.com/storybooks/storybook/issues/5436)

## What I did
I added autoprefixer to anguarl/cli webpack config.
Maybe it would be better to just add `// eslint-disable-line global-require` instead of added peerDependency. As autoprefixer is no direct dependency of an angular-cli project (it is in angular-devkit). And I think a missing peerDependency would then be reported by npm. But I did not dare adding a eslint disable without permission. 

## How to test
I did not find any ways to tests this automatically. If there is any please help.

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]
Looks good to me, 
WDYT @kroeder ?

File: app/angular/package.json [hunk 1]
```diff
@@ -51,7 +51,8 @@
     "@angular/platform-browser": ">=6.0.0",
     "@angular/platform-browser-dynamic": ">=6.0.0",
     "babel-loader": "^7.0.0 || ^8.0.0",
-    "zone.js": "^0.8.27"
+    "zone.js": "^0.8.27",
```
Comment (kroeder) [inline app/angular/package.json]: Sorry for the late response - due to a version update your zone.js change now produces a conflict. Can you fix this please? I think you just need to set `^0.8.29`

File: app/angular/package.json [hunk 2]
```diff
@@ -51,7 +51,8 @@
...
+    "autoprefixer": ">=8.1.0"
```
Comment (kroeder) [inline app/angular/package.json]: Is there a reason this has to be `>=8.1.0` instead of `^8.1.0` ?

Comment (alexan) [reply app/angular/package.json]: thats how angular-cli generated the dependeny in my project. I changed this now

**Comment by kroeder** [comment]
@ndelangen do we need to worry about these 4 chromatic changes?

**Comment by alexan** [comment]
I can squash the commits If necessary

**Review by kroeder — APPROVED** [review]

**Comment by shilman** [comment]
@alexan No squash commits please

**Pull request merged**